server/server.js

```diff
@@ -54,19 +54,19 @@ const server = function server(db) {
           return done(err);
         }
         // user not found
-        if (!user) {
+        if (!user[0]) {
           return done(null, false);
         }
         // wrong password
-        if (!myUsers.verifyPassword(user, password)) {
+        if (!myUsers.verifyPassword(user[0], password)) {
           return done(null, false);
         }
-        return done(null, { email: user.email });
+        return done(null, { email: user[0].email });
       });
     }
   ));
 
-  // Serialize & deserialize user
+  // Serialize & deserialize user NOTE: still dont know what it does
```

Comment:
serializeUser determines, which data of the user object should be stored in the session.

Also, we could use this with link saving.